archivers/unshield/Portfile

```diff
@@ -12,25 +13,23 @@ long_description \
 	Extract files from InstallShield-style cabinet files (usually named \
 	data1.cab, data1.hdr, data2.cab, etc), versions 5 and later
 
+homepage			https://github.com/twogood/unshield
```

Comment:
you should use the `github` PortGroup here as that will simplify things! Include that at the top and then you can do 
```
github.setup twogood unshield 1.4.3
github.tarball_from archives
```

that will allow you to remove the `homepage`, `master_sites`, and `distfiles` fields as they are set automatically.

Reply:
`github.setup` will also replace the `name` & `version` fields, so you can remove those alongside `homepage`, `master_sites`, and `distfiles`.

Reply:
good point @herbygillot - I thought that was kind-of obvious, but it doesn't hurt to mention explicitly ;)

Reply:
Thanks for the pointers. I struggled with setting up the github portgroup initially, as I missed the requirement to use `github.tarball_from archive`. It works now. 😃

Reply:
whether or not you use `github.tarball_from archive` should not matter - it should work regardless. It's just that we prefer to take the tarball from the `archives` instead of the commit hash.